### ISSUE TITLE
Usage poll

### ISSUE THREAD

**Issue by hirsivaja** [hirsivaja/plugin.video.areena#45]
The YLE is considering removing functionality from the API that would make this plugin not work anymore.
It would good to know how many people are using this plugin. Please respond below on how often you use the plugin on average.

I use the plugin:
[![](https://api.gh-polls.com/poll/01F1W19MCC55NQ1V5ZR62VV39T/Daily)](https://api.gh-polls.com/poll/01F1W19MCC55NQ1V5ZR62VV39T/Daily/vote)
[![](https://api.gh-polls.com/poll/01F1W19MCC55NQ1V5ZR62VV39T/Weekly)](https://api.gh-polls.com/poll/01F1W19MCC55NQ1V5ZR62VV39T/Weekly/vote)
[![](https://api.gh-polls.com/poll/01F1W19MCC55NQ1V5ZR62VV39T/Monthly)](https://api.gh-polls.com/poll/01F1W19MCC55NQ1V5ZR62VV39T/Monthly/vote)
[![](https://api.gh-polls.com/poll/01F1W19MCC55NQ1V5ZR62VV39T/Rarely%20or%20not%20anymore)](https://api.gh-polls.com/poll/01F1W19MCC55NQ1V5ZR62VV39T/Rarely%20or%20not%20anymore/vote)

**Comment by KasteluKannu** [comment]
I think there are a lot of people using the plugin, but not active on Github.

**Comment by hirsivaja** [comment]
> 
> 
> I think there are a lot of people using the plugin, but not active on Github.
That is true. Also this message will not reach all the users of the plugin, regardless if they use Github or not. But at least we should get some kind of minimum about the user base.

**Comment by AndersGustafsson** [comment]
True. Not much point in supporting it if no-one is using it. I for one would miss it and be forced to watch it on a tablet instead :( Any idea why YLE is changing the API?

You are also correct about GIT. We use SVN at work so even if I have a GIT account, I had to look really hard to find my credentials :)

**Comment by bitbooster** [comment]
Well, one way (a little intrusive, I have to admit) to poll the interest would be to push an update for the add-on that at that its first run would simply pop up a warning about the upcoming changes for the YLE API and the uncertain future of the add-on, with perhaps an invitation to go to a web page to answer a usage poll/survey.

**Comment by karsta62** [comment]
I definitely would be using this, but it stopped working already. Actually playback stopped. Everything else seems to work.
I don't have any smart TV so this plugin would be quite essential to me.
And it's true that not many people find this vote. Took a while to me as well when I started to google if I'm the only one having this playback issue.
Still not quite sure

**Comment by hirsivaja** [comment]
I guess it is just you :smile: 

You can create a new issue. Please list your os and kodi version. Also provide the actual error message from kodi logs.
Thanks.

**Comment by karsta62** [comment]
Ticket created. Thanks.

**Comment by bzoltan1** [comment]
Suosittelen lähettää palautetta Ylelle ->
tekninen.neuvonta@yle.fi

**Comment by suoranta** [comment]
I watch everything on head-mounted ("VR", e.g. Oculus) display (Kodi has native support for these), and I think more people have it on living room PCs. No more Areena for me. I did tell this to Yle, not surprised that they didn't react. They get our 160€/year or so "subscription fees" anyhow, so why bother with serving the customers. :-D

**Comment by Franz47** [comment]
Since I use Kodi/LibreElec on a Raspberry Pi behind the living room TV it was convenient to have the Areena plugin. Now I have to quit Kodi and boot the Raspbian partition and start a browser to watch Finnish TV. Is ist really so expensive for YLE to keep the interface? Would it ruin their finances completely?

**Comment by AndersGustafsson** [comment]
I have not seen any explanation from YLE why they dropped the open API, but at least did they flag their Tizen app for all the nordic countries so you can install it in Norway is you wish. I just hope that others, say SVT would follow suit.

**Comment by springjools** [comment]
I contacted YLE through email and they responded that they haven't shut down anything. They also asked what part of the API was in question.

**Comment by AndersGustafsson** [comment]
This is what they sent out this spring:
*** Hello, user of Yle API!

We at Yle have been taking a look at open interfaces based on their use. We
are disabling some of the lesser-used interfaces and focusing our
maintenance resources on the critical functions. For this reason, programme
data and stream distribution data interfaces will be removed from the Yle
APIs during the spring.

If your current API solution is of social or business importance to you or
your company, you can request access to an internal API whose interfaces
are different from the current implementation. At this stage, we are
accepting professional partners who use APIs to maintain services intended
for a broad user base as internal API users.

We are currently mapping out the use of open interfaces, and we hope that
you contact us via this form if the use of an internal API is necessary to
your operations:
https://forms.gle/hc6yFMyAvydgcq1JA 

We are aiming to provide access to the new interface to everyone whose
operations would be significantly hindered from not being able to use an
interface.

URLs of the APIs being disabled:

https://external.api.yle.fi/v1/programs/items.json 

https://external.api.yle.fi/v1/programs/items/{id}.json 

https://external.api.yle.fi/v1/series/items.json 

https://external.api.yle.fi/v1/series/items/{id}.json 

https://external.api.yle.fi/v1/programs/categories.json 

https://external.api.yle.fi/v1/programs/services.json 

https://external.api.yle.fi/v1/programs/services/{id}.json 

https://external.api.yle.fi/v1/programs/schedules.json 

https://external.api.yle.fi/v1/programs/schedules/now.json 

https://external.api.yle.fi/v1/programs/nowplaying/{service}.json 

https://external.api.yle.fi/v1/media/playouts.json 

https://external.api.yle.fi/v1/tracking/streamstart 


Regards,
Yle API team
api.apua@yle.fi
